lib/src/api.dart

```diff
@@ -167,14 +167,19 @@ class API {
   Future<dynamic> getWallet() async {
     Map<String, dynamic> resp = await _get('v2/wallets', private: true);
     if (resp != null && resp["data"] != null) {
+      Map<String, String> walletModules = Map<String, String>.from(resp['data']['walletModules']);
+      String dAIPointsManager = walletModules['DAIPointsManager'] != null
+          ? resp['data']['walletModules']['DAIPointsManager']
+          : null;
       return {
         "phoneNumber": resp["data"]["phoneNumber"],
         "accountAddress": resp["data"]["accountAddress"],
         "walletAddress": resp["data"]["walletAddress"],
         "createdAt": resp["data"]["createdAt"],
         "updatedAt": resp["data"]["updatedAt"],
-        "communityManager": resp['data']['walletModules']['CommunityManager'],
-        "transferManager": resp['data']['walletModules']['TransferManager'],
+        "communityManager": walletModules['CommunityManager'],
+        "transferManager": walletModules['TransferManager'],
+        "dAIPointsManager": dAIPointsManager,
```

Comment:
* typo: `dAIPointsManager` --> `daiPointsManager`
* also, maybe you can do:
```
"daiPointsManager": walletModules['DAIPointsManager'] || null 
```
and avoid saving it in a variable, so the code looks a bit cleaner